orte/mca/rmaps/seq/help-orte-rmaps-seq.txt

```diff
@@ -23,8 +23,10 @@ There are not enough slots available in the system to satisfy the %d slots
 that were requested by the application:
   %s
 
-Either request fewer slots for your application, or make more slots available
-for use.
+Either request fewer slots for your application, make more slots available
+for use. If oversubscription is intended, add --oversubscribe to the
+commandline.
```

Comment:
Nit pick: "command line" is two words, not one.

Reply:
Pushed a new commit with the fix, plus I also cleaned up the rest of the same help message.